### PR TITLE
Changes relevant for em_nion reader

### PR DESCRIPTION
**Pull request by mkuehbach**

### CODE REVIEW

File: contributed_definitions/nyaml/NXcircuit_board.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+doc: |
+  Circuit board with e.g. ADC and/or DAC electronic components.
+  
+  Currently used to store the settings of the so-called magboards used in
+  Nion electron microscopes but likely this could be a useful base class for
+  substantially more use cases where details at a deep technical instrument design
+  level are relevant or important.
+symbols:
+  doc: The symbols used in the schema to specify e.g. dimensions of arrays.
+NXcircuit_board:
```
Comment (sanbrock) [inline contributed_definitions/nyaml/NXcircuit_board.yaml:11]: if it is a specific board, maybe its name can also reflect that. In the future we may want to base it on a more generic NXcircuit from the spm branch

File: contributed_definitions/nyaml/NXdac.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+category: base
+doc: |
+  Digital-to-analog converter component/integrated circuit.
+symbols:
+  doc: The symbols used in the schema to specify e.g. dimensions of arrays.
+NXdac:
```
Comment (sanbrock) [inline contributed_definitions/nyaml/NXdac.yaml:6]: similarly to circuit_board, this can also be based on NXcircuit in the future

**Review by sanbrock — APPROVED** [review]
LGTM for now

File: contributed_definitions/nyaml/NXdac.yaml (hunk 2 above)
Comment (mkuehbach) [reply contributed_definitions/nyaml/NXdac.yaml:6]: Agreed

**Pull request merged**